Sources/KlaviyoCore/KlaviyoEnvironment.swift

```diff
@@ -150,7 +151,11 @@ public struct KlaviyoEnvironment {
             let notificationSettings = await UNUserNotificationCenter.current().notificationSettings()
             return PushEnablement.create(from: notificationSettings.authorizationStatus)
         },
-        getBackgroundSetting: { .create(from: UIApplication.shared.backgroundRefreshStatus) },
+        getBackgroundSetting: { .create(from: UIApplication.shared.backgroundRefreshStatus)
+        },
+        getBadgeAutoClearingSetting: {
+            Bundle.main.object(forInfoDictionaryKey: "Klaviyo_badge_autoclearing") as? Bool ?? true
```

Comment:
if we are unable to get the plist value we assume that they want to auto clear? I think this is fine but might be good to callout in the readme.